tomcat/src/main/scala/org/http4s/server/tomcat/TomcatBuilder.scala

```diff
@@ -5,23 +5,24 @@ package tomcat
 import cats.effect._
 import java.net.InetSocketAddress
 import java.util
+import java.util.concurrent.Executor
 import javax.servlet.http.HttpServlet
 import javax.servlet.{DispatcherType, Filter}
 import org.apache.catalina.{Context, Lifecycle, LifecycleEvent, LifecycleListener}
 import org.apache.catalina.startup.Tomcat
 import org.apache.catalina.util.ServerInfo
+import org.apache.coyote.AbstractProtocol
 import org.apache.tomcat.util.descriptor.web.{FilterDef, FilterMap}
 import org.http4s.server.SSLKeyStoreSupport.StoreInfo
 import org.http4s.servlet.{AsyncHttp4sServlet, ServletContainer, ServletIo}
 import org.log4s.getLogger
 import scala.collection.JavaConverters._
 import scala.collection.immutable
-import scala.concurrent.ExecutionContext
 import scala.concurrent.duration._
 
 sealed class TomcatBuilder[F[_]] private (
     socketAddress: InetSocketAddress,
-    private val executionContext: ExecutionContext,
+    externalExecutor: Option[Executor],
```

Comment:
As with Jetty, we want to shift blocking IO back to this.  It's going to hurt if it's internal.